app/assets/javascripts/autoform_form.js

```diff
@@ -19,14 +20,57 @@ $(document).ready(function() {
             return false;
         }
         else {
-            $('#autoform-error').hide();
-            $('#autoform-email').parent('.form-group').removeClass('has-error');
-            $('#autoform-domain').parent('.form-group').removeClass('has-error');
-            $('#autoform-form').hide();
-            $('#autoform-form-sent').show();
+            const formData = $(this).serialize();
+
+            $.post($(this).attr('action'), formData)
+                .done(function() {
+                    $('#autoform-error').hide();
+                    $('#autoform-form-error').hide();
+                    $('#autoform-email').parent('.form-group').removeClass('has-error');
+                    $('#autoform-domain').parent('.form-group').removeClass('has-error');
+                    $('#autoform-form').hide();
+                    $('#autoform-form-sent').show();
+                })
+                .fail(function() {
+                    $('#autoform-form-error').show();
+                });
 
             plausible('Registration', {props: {product: 'Autoform'}});
             return true;
         }
     });
+
+    const autoformError = document.getElementById('autoform-error');
```

Comment:
máme jQuery. Vyššie v kóde sa všade používa $ selector - je to kratšie a krajšie.